dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/CustomizationFilesServlet.java

```diff
@@ -33,113 +38,27 @@
-        BufferedReader in = null;
-        try {
-            File fileIn = physicalPath.toFile();
-            if (fileIn != null) {
-                inputStream = new FileInputStream(fileIn);
-
-                in = new BufferedReader(new InputStreamReader(inputStream));
-                String line;
-
-                StringBuilder responseData = new StringBuilder();
-                try (PrintWriter out = response.getWriter()) {
-
-                    while ((line = in.readLine()) != null) {
-                        responseData.append(line);
-                        out.println(line);
-                    }
-                }
-
-                inputStream.close();
-
-
-            } else {
-                /*
-                   If the file doesn't exist or it is unreadable we don't care
-                */
-            }
+    private static final Logger logger = Logger.getLogger(CustomizationFilesServlet.class.getSimpleName());
 
-        } catch (Exception e) {
-                /*
-                   If the file doesn't exist or it is unreadable we don't care
-                */
-        } finally {
-            IOUtils.closeQuietly(inputStream);
-            IOUtils.closeQuietly(in);
-        }
-
-    }
-
-    private String getFilePath(String fileTypeParam) {
-
-        if (fileTypeParam.equals(CustomizationConstants.fileTypeHomePage)) {
-
-            // Homepage
-            return settingsService.getValueForKey(SettingsServiceBean.Key.HomePageCustomizationFile);
-
-        } else if (fileTypeParam.equals(CustomizationConstants.fileTypeHeader)) {
-
-            // Header
-            return settingsService.getValueForKey(SettingsServiceBean.Key.HeaderCustomizationFile);
-
-        } else if (fileTypeParam.equals(CustomizationConstants.fileTypeFooter)) {
-
-            // Footer        
-            return settingsService.getValueForKey(SettingsServiceBean.Key.FooterCustomizationFile);
-
-        } else if (fileTypeParam.equals(CustomizationConstants.fileTypeStyle)) {
-
-            // Style (css)               
-            return settingsService.getValueForKey(SettingsServiceBean.Key.StyleCustomizationFile);
-
-        } else if (fileTypeParam.equals(CustomizationConstants.fileTypeAnalytics)) {
-
-            // Analytics - appears in head               
-            return settingsService.getValueForKey(SettingsServiceBean.Key.WebAnalyticsCode);
-
-        } else if (fileTypeParam.equals(CustomizationConstants.fileTypeLogo)) {
-
-            // Logo for installation - appears in header               
-            return settingsService.getValueForKey(SettingsServiceBean.Key.LogoCustomizationFile);
-        }
+    private static final Map<String, Key> SETTINGS_MAPPING = Initializer.createSettingsMapping();
 
+    private static final Set<Key> LOCALIZED_FILES_KEYS = Initializer.createSetOfLocalizedFilesKeys();
 
-        return "";
-    }
+    private static final String EN_LOCALE_CODE = Locale.ENGLISH.getLanguage();
```

Comment:
I'll approve pr, but do we even need this field? Can't we just go for explicit `Locale.ENGLISH.getLanguage()`?

Reply:
We could, but as ```Locale.ENGLISH.getLanguage()``` is not a constant but an accessor I think that it's better to store the value once (and that is used in two places in the class).